Domain.FSharp/Domain.FSharp.fsproj

```
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
```

Shouldn't this be [`netstandard2.1`](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) now that there [an LTS version](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/) out?

```suggestion
        <TargetFramework>netstandard2.1</TargetFramework>
```

---

The rest of the project is not compatible with netstandard 2.1. I will convert it to 2.1 when I'll finish the port.